vpr/src/util/vpr_utils.cpp

```diff
@@ -2200,6 +2210,21 @@ int get_physical_pin(t_physical_tile_type_ptr physical_tile,
     return result->second.pin;
 }
 
+int net_pin_tile_index(const ClusterNetId net_id, int net_pin_index) {
+    auto& cluster_ctx = g_vpr_ctx.clustering();
+
+    // Get the logical pin index of pin within it's logical block type
+    auto pin_id = cluster_ctx.clb_nlist.net_pin(net_id, net_pin_index);
+
+    return pin_tile_index(pin_id);
+}
+
+int pin_tile_index(const ClusterPinId pin) {
+    auto& place_ctx = g_vpr_ctx.placement();
+
+    return place_ctx.physical_pins[pin];
+}
+
```

Comment:
We probably want some comments describing what these are doing (probably at the relevant header declaration)

Reply:
Done